skel/problem/problem.yaml

```diff
@@ -1,4 +1,5 @@
-name: {%problemname%}
+name:
+  en: {%problemname%}
```

Comment:
This currently generates:
```
name:
  en: {'en': 'Test'}
```
Something should unwrap this somewhere, I think? :stuck_out_tongue:

Reply:
hmm this one is tricky... my best idea right now is to either undo the change or to ensure that new_problem always asks for the english name...

Reply:
It looks like `bt new_problem` asks for the English name already, sounds good to me :slightly_smiling_face: 

```
$ bt new_problem 
problem name (en): 
```